### PR TITLE
Add Reo.Dev Integration

### PR DESCRIPTION
**Pull request by vibhanshub**

### CODE REVIEW

**Comment by addisonschultz** [comment]
Looks great! 

One request - would you mind updating the folder name from `reo.dev` to `reo`? To avoid having `.` in the folder name—Additionally the service is just called "Reo" 

Code is looking good, we'll just wait to see where the 403 error is coming from before merging this in.

**Review by addisonschultz — DISMISSED** [review]
Looks great!

One request - would you mind updating the folder name from reo.dev to reo? To avoid having . in the folder name—Additionally the service is just called "Reo"

Code is looking good, we'll just wait to see where the 403 error is coming from before merging this in.

**Comment by addisonschultz** [comment]
Hi @vibhanshub - I've had to update the configuration and manifest based on some updates coming to sites. The context is mentioned in this PR (https://github.com/GitbookIO/integrations/pull/453/files)

This should be all that is needed, but can you review my last commit, and **re-publish this integration if everything looks okay**? We can keep the PR open, but just need to update the published version.

Let me know if you have any questions

File: integrations/reo/gitbook-manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: reodotdev
+title: Reodotdev
```
Comment (taranvohra) [inline integrations/reo/gitbook-manifest.yaml:2]:
```suggestion
title: Reo.dev
```

File: integrations/reo/gitbook-manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+organization: d8f63b60-89ae-11e7-8574-5927d48c4877
+contentSecurityPolicy:
+    script-src: static.reo.dev www.ipapi.co;
+summary: |
+    # Overview
+    Reo.Dev is the Business to Developer (B2D) ABM platform that helps you find revenue opportunity from developer activity.
+    This integration lets you add the Reo.Dev Javascript on your published GitBook site to seamlessly analyze developer intent through any interactions with your content on GitBook.
+
+    # How it works
+    The integration injects the Reodotdev script on your page, using the configured client ID,
```
Comment (taranvohra) [inline integrations/reo/gitbook-manifest.yaml:26]:
```suggestion
    The integration injects the Reo.dev script on your page, using the configured client ID, 
```

**Review by taranvohra — APPROVED** [review]

**Pull request merged**